### PR TITLE
queue: unify naming

### PR DESCRIPTION
**Pull request by BusyJay**
Change simple queue to work stealing queue, and use `TaskInjector`
instead of `QueueInjector`, `LocalQueue` instead of `QueueLocal`.

### CODE REVIEW

**Comment by sticnarf** [comment]
I'm a bit worried that when we have a `work_stealing` queue, people may misunderstand the other queue is not work stealing .

**Comment by BusyJay** [comment]
Do you have any recommendation? How about `SingleLevel`?

**Comment by sticnarf** [comment]
`SingleLevel` sounds fine :)

File: src/queue.rs [hunk 1]
```diff
@@ -27,7 +27,7 @@ pub trait TaskCell {
 pub struct TaskInjector<T>(InjectorInner<T>);
 
 enum InjectorInner<T> {
-    WorkStealing(work_stealing::TaskInjector<T>),
+    WorkStealing(single_level::TaskInjector<T>),
```
Comment (sticnarf) [inline src/queue.rs]: Enum variant name needs updating.

**Review by sticnarf — APPROVED** [review]
LGTM

**Review by AndreMouche — APPROVED** [review]
LGTM

**Pull request merged**